changelog.yaml

```diff
@@ -30,6 +30,28 @@
       Feature flags can be configured via Helm Chart.
     pullRequestNumber: 178
 
+  - type: enhancement
+    impact: minor
```

Comment:
@rinckm For me this change does not look compatible. What happens if `networkPolicy` is not changes to `networkPolicies`.

Reply:
This will still work.

See fallback to old syntax:
 charts/steward/templates/config-pipelineruns-network-policies.yaml  L80 + L104ff